cpp/src/com/google/idea/blaze/cpp/CompilerInfo.java

```diff
@@ -0,0 +1,21 @@
+package com.google.idea.blaze.cpp;
+
+public class CompilerInfo {
```

Comment:
I think we should remove this class and maybe we can call the map `targetToCompilerVersionStrMap` and this way it will be clear that `targetToCompilerVersionStrMap.get(target)` calls return compiler version strings.